sdk/rust/oak_tests/src/lib.rs

```diff
@@ -46,42 +47,35 @@ pub fn compile_rust_wasm(
     module_wasm_file_name: &str,
     profile: Profile,
 ) -> anyhow::Result<Vec<u8>> {
-    // Use a separate target dir for Wasm build artifacts. The precise name is not relevant, but it
+    // Use a separate output dir for Wasm build artifacts. The precise name is not relevant, but it
     // should end with `target` so that it gets automatically ignored by our `.gitignore`.
-    let target_dir = PathBuf::from("oak_tests/target");
+    let out_dir = "oak_tests/target";
 
     let mut args = vec![
+        // `--out-dir` is unstable and requires `-Zunstable-options`.
+        "-Zunstable-options".to_string(),
         "build".to_string(),
-        format!(
-            "--target-dir={}",
-            target_dir.to_str().expect("invalid target dir")
-        ),
         "--target=wasm32-unknown-unknown".to_string(),
         format!("--manifest-path={}", manifest_path),
+        format!("--out-dir={}", out_dir),
     ];
-    let profile_str = match profile {
-        Profile::Release => {
-            args.push("--release".to_string());
-            "release".to_string()
-        }
-        Profile::Debug => "debug".to_string(),
-    };
+    if profile == Profile::Release {
```

Comment:
Should these be kept in sync with runner? if so, could you add a comment here and also in runner?

Reply:
Added a comment